spec/features/data_access/data_download_spec.rb

```diff
@@ -209,6 +209,9 @@
       wait_for_xhr
     end
 
+    it 'does not display a header erroneously declaring success' do
+      expect(page).should_not have_content("Success!")
+    end
```

Comment:
Need to test the cases when "Request in Progress" shows up and when "Request in Progress" disappears. 
 - an order in different states ("Creating" - show, "Submitting" - show, "Complete" - hide, etc.)
 - a service order in different states (similar, see above.)